test/e2e/suite_test.go

```diff
@@ -101,6 +102,11 @@ func beforeTest(t *testing.T, namespace, fixtureDir string, f *framework.E2eFram
 		return err
 	}
 
+	if err := f.DeployK8sClientConfigs(namespace); err != nil {
```

Comment:
`f.DeployK8sClientConfigs` fails because the CRDs are deployed in `f.DeployK8ssandraOperator`. You'll need to deploy the CRDs before this call. I would also move up this block:

```go
if err := f.WaitForCrdsToBecomeActive(); err != nil {
		t.Log("failed waiting for CRDs to become active")
		return err
}
```